README.md

```
@@ -57,7 +57,7 @@ The main way to install EVOKS and its depending services is to build and run the
 |-------------------|--------------------|----------------------------------------------------------------------------------------------------------------------------|--------------------|---|
 | INSTANCE_NAME     | defaultinstance    | Prefix of docker container names                                                                                           | no                 |   |
 | PROXY_PORT        | 9000               | Port where the proxy is located                                                                                            | no                 |   |
-| PUBLICURL         | http://localhost   | (public) base URL under which the services will be reached                                                                 | no                 |   |
+| PUBLICURL         | http://localhost   | (public) base URL under which the services will be reached (adding "http://" resp. "https://" is mandatory)                | no                 |   |
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)